spec/system/candidate_interface/candidate_entering_degrees_spec.rb

```diff
@@ -86,7 +86,7 @@ def when_i_fill_in_my_undergraduate_degree
 
     choose t('application_form.degree.grade.first.label')
 
-    fill_in t('application_form.degree.award_year.label'), with: '2009'
+    fill_in t('application_form.degree.award_year.label'), with: ' 2009'
```

Comment:
Does this space need a comment, to avoid future developers deleting?
Or a variable name "year_with_preceding_space"

Reply:
Yeah, that makes sense! Will update this.